### DISCUSSION TITLE
Fail2ban won't work after updating or redeploying Vaultwarden

### DISCUSSION THREAD

**Discussion by Chippel** [dani-garcia/vaultwarden#3682] [Q&A]
### Subject of the issue
I've set up Vaultwarden successfully via Docker and Nginx reverse proxy manager. Also, I set up the logfile environment variable and I find the logfile it in Linux. I followed the Fail2ban manual and everything works as expected. I get banned and the website will not be accessible anymore if I enter too many bad credentials.

Unfortunately after updating it won't work anymore. I can still see the logfile and it gets updated but I won't get banned via Fail2ban anymore.

I did "update" the container as followed:

1) went into Portainer and removed the container in the container tab.
2) removed the image in the "images" tab.

After that I just created a new container with all the same settings. Settings were kept cuz I could access my admin panel and SMTP settings were still available.

### Deployment environment

I did set up manually via Portainer. I set the name, gave the image link and I also created a volume which is called vw-data and which points to /data/

* vaultwarden version:

manual Portainer way by clicking the buttons.

* Clients used: *
Android, Windows 11.

* Reverse proxy and version: 
Nginx Proxy Manager

EDIT: seems like the vaultwarden.log file won't get updated after updating. Do I have to set the logpath again when I update? That does not seem to be convinient.

**Comment by nomad2246** [comment]
It works perfectly for me.

To update from portainer is as simple as go to the stack, editor tab, "update the stack" button and check the "Re-pull image and redeploy" option.
Then if you want you go to the image section and delete the old one.
